paywall/src/unlock.js/index.ts

```diff
@@ -1,16 +1,58 @@
 import startup from './startup'
 import '../paywall-builder/iframe.css'
-import { UnlockWindow } from '../windowTypes'
+import { UnlockWindowNoProtocolYet } from '../windowTypes'
+import StartupConstants from './startupTypes'
+
+declare const process: {
+  env: {
+    UNLOCK_ENV: 'dev' | 'test' | 'staging' | 'prod'
+    DEBUG: any
+    PAYWALL_URL: string
+    USER_IFRAME_URL: string
+  }
+}
+
+const constants: { [key: string]: StartupConstants } = {
```

Comment:
I am a bit worried by this, but I guess we can move it to its own file and only use the ones we need there?

Reply:
these are the shared constants that are needed, and the idea was to construct them in `index.ts`, similarly to how the data iframe does things. I don't think it matters much where they live, so a separate file would work fine